server/utils/helpers.js

```diff
@@ -0,0 +1,57 @@
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcrypt';
+import Joi from 'joi';
+import env from '../config/env-config';
+
+/**
+ * Synchronously sign the given payload into a JSON Web Token string
+ * @param {string | object | Buffer} payload
+ * @returns {string}
+ */
+export const generateToken = payload => jwt.sign(payload, env.SECRET);
```

Comment:
No expiry?

Reply:
I'll change the function to also accept an `expiresIn` argument.